src/wire/libevent_callbacks.cpp

```diff
@@ -211,5 +213,36 @@ void StateMachine(LibeventSocket *conn) {
   }
 }
 
+/**
+ * Stop signal handling
+ */
+void Signal_Callback(UNUSED_ATTRIBUTE evutil_socket_t fd,
```

Comment:
Instead of polluting the global namespace, can you make these static functions inside of a utility class?